es-design-system/pages/molecules/es-video.vue

```diff
@@ -0,0 +1,51 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div>
+        <h1>
+            Video
+        </h1>
+        <p>
+            Shows the embed video when the cover is clicked.
```

Comment:
Since our URL parsing logic assumes the provided embed URL is a YouTube embed URL, I'd recommend making this a little more specific, like "Shows a YouTube video when the provided cover image is clicked."